docs/guides/multicollateral-warp-route.mdx

```diff
@@ -0,0 +1,178 @@
+
+## Liquidity Provider
+
+Currently, Warp Routes don’t have an explicit liquidity provider interface that enables local deposits/withdrawals. However, LPs can manually manage liquidity using the [Hyperlane CLI](https://www.npmjs.com/package/@hyperlane-xyz/cli) to interact with the Warp Routes in the [Hyperlane registry](https://github.com/hyperlane-xyz/hyperlane-registry).
+
+:::warning
+The stopgap procedure defined below requires at least one synthetic chain to exist within the warp route topology.
+:::
+
+- To inspect a Warp Route’s topology, use the `warp read` command:
+
+```
+hyperlane warp read --symbol ETH
+
+ethereum:
+    type: native
+    ...
+base:
+    type: native
+    ...
+bsc:
+    type: synthetic
+    ...
+```
+
+- To send a transfer (`transferRemote`) on a Warp Route, use the `warp send` command:
+
+```
+hyperlane warp send \
+	--symbol ETH \
+	--origin base \
+    --destination bsc \
+	--amount <AMOUNT> \
+	--recipient <ADDRESS>
+```
+
+### Depositing Liquidity
+
+LPs can deposit collateral via a `transferRemote` where:
+
+- `destination` domain is a chain where the warp route has a `synthetic` type
+- `recipient` address is controlled by the LP
+- `amount` is liquidity denominated in the `origin` chains `collateral` token
+
+```mermaid
+flowchart
+	LP((Liquidity Provider))
+
+	subgraph arbitrum
+		AWR[USDC Collateral Warp Route]
+		AUSDC[USDC]
+	end
+
+	subgraph mychain
+		MWR[USDC Synthetic Warp Route]
+	end
+
+	AWR -- "transferFrom(LP, amount)" --> AUSDC
+	LP -. "amount" .-> AWR
+	LP -- "transferRemote(newchain,<br>LP, amount)" --> AWR
+
+	AWR -. "{LP, amount}" .-> MWR
+	MWR -. "amount" .-> LP
+```
+
+This can be done for providing liquidity on many collateral chains and representing a claim on each collateral with a single synthetic asset balance.
+
+### Withdrawing Liquidity
+
+LPs can withdraw via a `transferRemote` where
+
+- `destination` domain is a chain where the warp route is a `collateral` type
+- `recipient` address is controlled by the LP
+- `amount` is denominated in the `destination` chains `collateral` token
+
+```mermaid
+flowchart
+	LP((Liquidity Provider))
+
+	subgraph arbitrum
+		AWR[USDC Collateral Warp Route]
+		AUSDC[USDC]
+	end
+
+	subgraph mychain
+		MWR[USDC Synthetic Warp Route]
+	end
+
+	LP -- "transferRemote(arbitrum,<br>LP, amount)" --> MWR
+	LP -. "amount" .-> MWR
+
+	MWR -. "{LP, amount}" .-> AWR
+	AWR -- "transfer(LP, amount)" --> AUSDC
+	AWR -. "amount" .-> LP
+```
+
+## Future: Admin-Controlled Liquidity Rebalancing
+
+Depending on the collateral asset and the pair of chains, several different rebalancing paths may be available. For example, CCTP (Circle’s Cross-Chain Transfer Protocol) can efficiently move USDC between Base and Arbitrum. Other assets and pairs may require more complex rebalancing paths (eg L2 withdrawals+deposits). This approach requires a permissioned rebalancer to move the collateral over trusted paths.
```

Comment:
im not sure how we want to hint at this publicly but would be good to check with @nambrot

Reply:
got it - yeah can remove the example here

Reply:
on second review - think we should remove the future section. made the change